src/drivers/dw/dma.c

```diff
@@ -336,6 +337,67 @@ static inline void dw_update_bits(struct dma *dma, uint32_t reg, uint32_t mask,
 	io_reg_update_bits(dma_base(dma) + reg, mask, value);
 }
 
+static inline void dw_dma_interrupt_mask(struct dma *dma, int channel)
```

Comment:
I'd rather not use "inline" especially in .c. The modern compilers should be smart enough to decide on that.